test/functional/activity-window-host-test.js

```diff
@@ -715,7 +715,7 @@ describes.realWin('ActivityWindowRedirectHost', {}, env => {
       const request = {
         requestId: 'request1',
         returnUrl: 'https://example.com/opener',
-        args: {a: 1},
+        args: {a: 1, text: '40%'},
```

Comment:
Keep existing test as well?

Reply:
It's the existing one. Just added to it.